commit 2d1e8b3f1acc36101dbe6c5fc14e88e3f6af0b1c
Author: Wanpeng Li <liwanp@linux.vnet.ibm.com>
Date:   Wed Sep 11 14:23:00 2013 -0700

    mm/hwpoison-inject.c: change permission of corrupt-pfn/unpoison-pfn to 0200
    
    Hwpoison injection doesn't implement read method for
    corrupt-pfn/unpoison-pfn attributes:
    
    # cat /sys/kernel/debug/hwpoison/corrupt-pfn
    cat: /sys/kernel/debug/hwpoison/corrupt-pfn: Permission denied
    # cat /sys/kernel/debug/hwpoison/unpoison-pfn
    cat: /sys/kernel/debug/hwpoison/unpoison-pfn: Permission denied
    
    This patch changes the permission of corrupt-pfn/unpoison-pfn to 0200.
    
    Signed-off-by: Wanpeng Li <liwanp@linux.vnet.ibm.com>
    Reviewed-by: Naoya Horiguchi <n-horiguchi@ah.jp.nec.com>
    Cc: Andi Kleen <andi@firstfloor.org>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

diff --git a/mm/hwpoison-inject.c b/mm/hwpoison-inject.c
index 3a61efc518d5..afc2daa91c60 100644
--- a/mm/hwpoison-inject.c
+++ b/mm/hwpoison-inject.c
@@ -88,12 +88,12 @@ static int pfn_inject_init(void)
 	 * hardware status change, hence do not require hardware support.
 	 * They are mainly for testing hwpoison in software level.
 	 */
-	dentry = debugfs_create_file("corrupt-pfn", 0600, hwpoison_dir,
+	dentry = debugfs_create_file("corrupt-pfn", 0200, hwpoison_dir,
 					  NULL, &hwpoison_fops);
 	if (!dentry)
 		goto fail;
 
-	dentry = debugfs_create_file("unpoison-pfn", 0600, hwpoison_dir,
+	dentry = debugfs_create_file("unpoison-pfn", 0200, hwpoison_dir,
 				     NULL, &unpoison_fops);
 	if (!dentry)
 		goto fail;